packages/nimbus/package.json

```diff
@@ -34,8 +34,18 @@
       ]
     }
   },
+  "dependencies": {
+    "@react-aria/optimize-locales-plugin": "^1.1.4",
```

Comment:
this can probably be a dev dependency as well, since it's only used during build time w/vite